lib/AuditoManager.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:audioplayers/audioplayers.dart';
+import 'package:path/path.dart';
+import 'package:sqflite/sqflite.dart';
+
+class AudioManager {
+  final AudioPlayer _audioPlayer = AudioPlayer();
+  Database? _database;
+  bool isPlaying = false;
+  bool loop = false;
+  double volume = 1.0;
+
+  Future<void> initializeDatabase() async {
+    String path = join(await getDatabasesPath(), 'mindfulness_audio.db');
+    _database = await openDatabase(
+      path,
+      version: 1,
+      onCreate: (db, version) {
+        db.execute("CREATE TABLE IF NOT EXISTS sessions (id INTEGER PRIMARY KEY, name TEXT)");
+        db.execute("CREATE TABLE IF NOT EXISTS cues (id INTEGER PRIMARY KEY AUTOINCREMENT, session_id INTEGER, time_sec INTEGER, message TEXT, FOREIGN KEY (session_id) REFERENCES sessions (id))");
+        db.execute("CREATE TABLE IF NOT EXISTS preferences (id INTEGER PRIMARY KEY, volume REAL)");
+      },
+    );
+    print("Database initialized.");
+
+    // Initialize preferences if not set
+    final prefs = await _database?.query('preferences');
+    if (prefs == null || prefs.isEmpty) {
+      await _database?.insert('preferences', {'volume': 1.0}); // default volume
+    } else {
+      volume = prefs[0]['volume'] as double;
+    }
+  }
+
+  Future<void> initializeAudioSession(int sessionId, String name) async {
+    await _database?.insert(
+      'sessions',
+      {'id': sessionId, 'name': name},
+      conflictAlgorithm: ConflictAlgorithm.ignore,
+    );
+    print("Session $sessionId initialized with name: $name.");
+  }
+
+  Future<void> playAudio(String audioFile, {bool loop = false}) async {
+    this.loop = loop;
+    isPlaying = true;
+    await _audioPlayer.setVolume(volume);
+    await _audioPlayer.play(AssetSource(audioFile), isLocal: true);
+    _audioPlayer.onPlayerComplete.listen((_) {
+      if (loop) {
+        playAudio(audioFile, loop: true);
+      } else {
+        isPlaying = false;
+      }
+    });
+    print("Playing audio: $audioFile");
+  }
+
+  Future<void> pauseAudio() async {
+    await _audioPlayer.pause();
+    isPlaying = false;
+    print("Audio paused.");
```

Comment:
1. Make the `print` statements only execute if the app isn't in a release/production environment. They can be useful for functions that interact with the database, but it's kind of just bloat with something simple like pausing the audio.
2. Why not use `debugPrint`?

Reply:
I can't